### ISSUE TITLE
Not able to pin to a lower version

### ISSUE THREAD

**Issue by royjit** [awslabs/aws-sdk-swift#545]
## Description
Compilation error happens if I try to pin aws-sdk-swift to a lower version.

## Context

I am trying to pin aws-sdk-swift to a particular version because I am not ready to take in the new version features in my app. 

## Environment

* AWS Swift SDK version used: 0.1.3
* Operating System and version (iOS/macOS/Linux): macOS
* Xcode version (if applicable): 13.2.1
* Swift compiler version (if applicable):
* Swift Tools version (if applicable):

## Reproduction Steps

1. Add the dependency on aws-sdk-swift with pinning the version to `0.1.3`
```
.package(
            name: "AWSSwiftSDK",
            url: "https://github.com/awslabs/aws-sdk-swift",
            .exact("0.1.3")
        )
```
2. Build the app
3. See the compiler error

## Observed Behavior

Compiler error

Notice that the version shown for AwsCrt and ClientRuntime is 0.2.0 ie the latest version. 

## Expected Behavior

Should compile without any error

**Comment by royjit** [comment]
Tried running `swift package clean && swift build` See the logs below, aws-sdk-swift is resolved to 0.1.3 but the other two packages picks up 0.2.0 

```
Computed https://github.com/awslabs/aws-sdk-swift at 0.1.3 (6.24s)
Computing version for https://github.com/awslabs/smithy-swift.git
Computed https://github.com/awslabs/smithy-swift.git at 0.2.0 (0.56s)
Computing version for https://github.com/awslabs/aws-crt-swift.git
Computed https://github.com/awslabs/aws-crt-swift.git at 0.2.0 (0.84s)
```

**Comment by kneekey23** [comment]
Closing since @royjit fixed :)
